ui/src/app/icons/edit.spec.tsx

```diff
@@ -18,6 +18,7 @@ describe('EditIconComponent', () => {
   beforeEach(() => {
     props = {
       disabled: false,
+      enableHoverEffect: true,
```

Comment:
What does this do?

Reply:
I added `enableHoverEffect` as a prop to the edit component so I can toggle if I want that behavior or not.

The test needed to be updated with the new prop parameters.